apps/server/src/modules/files-storage/controller/dto/file-storage.response.ts

```diff
@@ -0,0 +1,35 @@
+/* istanbul ignore file */
```

Comment:
maybe we should name the file `filerecord.response.ts`? (would fit the entity more).

Reply:
it can have many response types